### ISSUE TITLE
Animation fails for the same test when placed in two different projects

### ISSUE THREAD

**Issue by ppeyro** [PredixDev/px-context-browser#54]
## Steps to reproduce the problem:

- Used the same context browser test case.
- Placed on in Context browser directory and a copy of the file in a different project(timeseries).
- bower link the timeseries directory to the context browser. 
- When I do a gulp serve and server the test file, context browser does render properly.
- If you select the item (up arrow icon), it seems to work fine.
- But if you select the open arrow to open the children panel, it results in a "cancel" error in the console log and the context browser instance becomes unresponsive at that point.

## Environment:

CB: Redesign branch/latest pull
Browser: Chrome59

### webanimation versions context browser vs. timeseries projects:

![screen shot 2017-07-07 at 3 45 20 pm](https://user-images.githubusercontent.com/10835611/27979569-bdd3f57e-632b-11e7-9385-43e8526e760b.png)

### Error in Timeseries Project

![cancel_error_in_ts](https://user-images.githubusercontent.com/10835611/27979601-02189dca-632c-11e7-8a63-fd9aa759f26a.png)

### No error in Context Browser Project

![no_error_in_cb](https://user-images.githubusercontent.com/10835611/27979616-176322ea-632c-11e7-8233-88d8e6a1e624.png)

## Code example:
```
<!doctype html>

<html>
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <title>CB</title>
        <!-- Requires Webcomponents.js polyfill is provided by the page for browsers that don't support html imports -->
        <script src="../webcomponentsjs/webcomponents-lite.min.js"></script>
        <!-- Import custom element. Note: see comment about relative paths to dependencies in the *.html file referenced below -->
        <link rel="import" href="../px-theme/px-theme-styles.html">
        <!-- <link rel="import" href="../px-dark-theme/px-dark-theme-styles.html"> -->
        <style include="px-theme-styles" is="custom-style"></style>

        <link rel="import" href="../iron-ajax/iron-ajax.html"/>
        <link rel="import" href="px-context-browser.html">
    </head>

    <dom-module id="demo-context-browser">
      <body style="padding:30px; ">
    <!-- <body style="padding:30px; background-color: rgb(27,42,51);"> -->
      <template id="CBtest1" is="dom-bind">

      <h2>NEW CB </h2>

      <style> px-context-browser {--px-context-browser-panel-width:500px} </style>

      <px-context-browser
          items="{{data}}"
          collapse-all
          collapse-with-icon
          open-arrow
          has-favorites
          has-filter
          >
      </px-context-browser>

      <iron-ajax auto url="padi_cb1.json" last-response="{{data}}"></iron-ajax>
    </dom-module>
  </template>
  </body>
</html>


/cc @davidrleonard

**Comment by davidrleonard** [comment]
Fixed as of 63d123746b924559b9c54703fdf06c4b5dc61cb5, when we switched from the WebAnimations API to good old fashioned CSS animations.
